internal/driver/interactive.go

```diff
@@ -64,6 +54,9 @@ func interactive(p *profile.Profile, o *plugin.Options) error {
 			}
 		}
 
+		// Get all groups in pprofVariables to allow for clearer error messages.
+		groups := groupOptions()
```

Comment:
Move this out of the for loop to avoid calling it on every input line?

Reply:
Done.